manifests/init.pp

```diff
@@ -121,16 +133,29 @@
   }
 
   if $use_fqdn_real == true {
-    @@host { $::fqdn:
-      ensure       => $fqdn_ensure,
-      host_aliases => $my_fqdn_host_aliases,
-      ip           => $fqdn_ip,
+    if $export_tag == undef {
+      @@host { $::fqdn:
```

Comment:
I did try that at first, but it failed one test: Ruby 1.8.7 + Puppet Gem 3 w/ Future Parser (https://travis-ci.org/ghoneycutt/puppet-module-hosts/jobs/164903651)
Is there a better approach that will still pass that test?